Jahsie/template/memory-gui.css

```diff
@@ -0,0 +1,111 @@
+/* CSS for GUI...*/
+@import url(http://fonts.googleapis.com/css?family=Lato:400,900);
+
+html {
+	font-size: 12px;
+}
+
+@media screen and (min-width: 900px){
+  html{
+    font-size: 13px;
+  }
+}
+
+
+body {
+	font-family: 'Lato', sans-serif;
+	color: grey;
+	margin: 0px;
+	height: 100%
+}
+
+h1 {
+	font-weight: 900;	
+}
+
+#intro {
+	box-sizing: border-box;
+	position: absolute;
+	display: inline-block;
+	width: 20%;
+	float: left;
+	text-align: center;
+	height: 100%;
+	padding: 3% 0px 0px 3%;
+}
+
+.quote {
+	font-style: italic;
+	font-size: 1rem;
+	vertical-align: bottom;
+/*	FIX THIS!*/
+	margin-top: 100%;
+	position: relative;
```

Comment:
This doesn't work here because `relative` applies in relation to the containing element. The div that this paragraph is in is positioned absolutely, and the paragraph just positions itself relative to that. It's the containing div that's making the problem.